scripts/bootstrap.sh

```diff
@@ -102,7 +108,7 @@ direnv_shell_integration() {
 		EOF
 		;;
 	*)
-		fail "Don't know how to setup for shell $SHELL. checkout https://direnv.net/docs/hook.html"
+		echo "Don't know how to setup for shell $SHELL. checkout https://direnv.net/docs/hook.html"
```

Comment:
Note for anyone wondering about this change in future - the function `fail` didn't exist, it was a copy-paste error this PR cleans up